ui/components/CustomToolbarSelect.js

```diff
@@ -121,51 +118,55 @@ class CustomToolbarSelect extends React.Component {
             <CompareArrowsIcon className={[classes.icon, classes.inverseIcon].join(" ")} />
```

Comment:
These comments should also be deleted if we are deleting comments as in ui/components/CustomTableFooter.js